umm-spec-lib/src/cmr/umm_spec/test/expected_conversion_util.clj

```diff
@@ -13,6 +14,23 @@
   [cmr.umm-spec.url :as url]
   [cmr.umm-spec.util :as su]))
 
+(def coll-progress-enum-list
+  "The enum list for CollectionProgress in v1.10. that could be converted from 
+   all formats except for DIF10"
+  (set ["PLANNED" "ACTIVE" "COMPLETE" "NOT PROVIDED" "NOT APPLICABLE"]))
+
+(defn expected-coll-progress
```

Comment:
Not sure why this is needed. When doing conversions, we should always be converting to valid UMM, so we shouldn't need to do an additional conversion.